src/main/webapp/css/jenkins-style.css

```
@@ -22,6 +46,12 @@
 /* Font awesome icons float right                                                                                      */
 /* ------------------------------------------------------------------------------------------------------------------- */
 
+.card {
+    background-color: #fff;
+    background-color: var(--background);
+    border: var(--input-border);
```

Duplicate assignment:
```suggestion
    background-color: #fff;
    color: var(--text-color);
```
I also added `color` and removed `border`, otherwise there are no borders shown anymore